mailjet_rest/client.py

```diff
@@ -12,17 +12,18 @@
 
 
 class Config(object):
-    API_URL = 'https://api.mailjet.com/'
+    DEFAULT_API_URL = 'https://api.eu.mailjet.com/'
     API_REF = 'http://dev.mailjet.com/email-api/v3/'
     version = 'v3'
     user_agent = 'mailjet-apiv3-python/v' + get_version()
 
-    def __init__(self, version=None):
+    def __init__(self, version=None, api_url=None):
         if version is not None:
             self.version = version
+        self.api_url = api_url or self.DEFAULT_API_URL
 
     def __getitem__(self, key):
-        url = self.API_URL[0:]
+        url = self.api_url[0:]
```

Comment:
The slicing does not really do anything so it's better off without it:
```python
In [1]: 'https://api.eu.mailjet.com/'[0:]
Out[1]: 'https://api.eu.mailjet.com/'
```

Reply:
Indeed you are correct. Slicing has been removed.